games/TicTacToe/project/tictactoe-contract/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[project]
+name = "tictactoe-contract"
+version = "0.0.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+fuels = { version = "0.33.0", features = ["fuel-core-lib"] }
```

Comment:
Though this is by no means a strict requirement, I would like to point out that all apps use `fuels 0.34.0`

Reply:
Oh right, I've mentioned that in our meet but I've forgotten about this